### ISSUE TITLE
silent format is missing

### ISSUE THREAD

**Issue by abrekhov** [aelsabbahy/goss#656]
**Describe the bug**
I'm trying to make goss validate with -f silent and getting such output:
`Error: bad output format: silent`

**How To Reproduce**
```
docker run -ti alpine sh
apk add curl
echo hello > hello.txt
goss a file hello.txt
goss v -f silent
```

**Expected Behavior**
Exit status 0 and no error

**Actual Behavior**
`Error: bad output format: silent`

**Environment:**
 - Version of goss = 0.3.14
 - OS/Distribution version (if applicable)

**Comment by aelsabbahy** [comment]
@petemounce FYI. Also, if you have a chance take a  quick look at #657, pretty sure that's the fix.. just want to make sure I'm not missing anything with the refactor.

**Comment by aelsabbahy** [comment]
@abrekhov I'll try to get the fix released as soon as possible, in the meantime if you don't need the v0.3.14 features, v0.3.13 `silent` correctly works

**Comment by aelsabbahy** [comment]
Fixed in `0.3.15`

Thanks for reporting this!
